### PR TITLE
feat(Dramacool): more domain support

### LINKED ISSUE #6486: Fixing Dramacool link that is being used at the moment
### Presence name

Dramacool

### Description

Hello, I have noticed that whenever I load up dramacool, and I enable the presence on pre-mid for it. It never displays on my discord and I feel like the problem is the website that it is linked. I believe the current Dramacool presence is linked to this site 
image

Only if I enter the website above it will show dramacool on discord, but the problem is it's an outdated website

While the updated site link is this one on the bottom, I am hoping it can be changed to
image
I say this link is the updated one because I have searched up dramas and found them through the search bar, while the one on the first screenshot, I couldn't find the dramas or they are not up to date.

I hope the website link can be changed to the  2nd screenshot I have shown, thank you in the future if this gets fixed.

### Alternatives

_No response_

### Additional context

https://www.dramacool9.co/

Updated website link I am hoping can be added, instead of the old one.

### PR DESCRIPTION
**Pull request by DarkVillager**
## Description 

Add more support for different domain their elements. E.g. [new site ](https://www1.dramacool.ac/)
Resolves #6486

## Acknowledgements
- [X] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [X] I linted the code by running `yarn format`
- [X] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

**Review by EncryptedDev — APPROVED** [review]

File: websites/D/DramaCool/presence.ts [hunk 1]
```diff
@@ -98,11 +100,7 @@ presence.on("UpdateData", async () => {
 			presenceData.buttons = [
 				{
 					label: strings.viewEpisode,
-					url: document.URL,
-				},
-				{
-					label: strings.viewSeriesButton,
-					url: document.querySelector<HTMLAnchorElement>("div.category a").href,
```
Comment (Slowlife01) [inline websites/D/DramaCool/presence.ts:105]: Why is this removed?

Comment (DarkVillager) [reply websites/D/DramaCool/presence.ts:105]: It doesn’t work anymore

Comment (Slowlife01) [reply websites/D/DramaCool/presence.ts:105]: It's this very element

File: websites/D/DramaCool/presence.ts [hunk 2]
```diff
@@ -46,15 +45,13 @@ presence.on("UpdateData", async () => {
...
 		showButtons = await presence.getSetting<boolean>("buttons"),
-		{ pathname, search } = document.location;
+		{ pathname, search, href } = document.location;
 
 	if (oldLang !== newLang || !strings) {
 		oldLang = newLang;
 		strings = await getStrings();
 	}
 
-	if (pathname.includes("running-man")) presenceData.largeImageKey = "rm";
```
Comment (Slowlife01) [inline websites/D/DramaCool/presence.ts:56]: This is intentional

File: websites/D/DramaCool/presence.ts [hunk 3]
```diff
@@ -74,8 +71,13 @@ presence.on("UpdateData", async () => {
...
 	} else if (pathname.match("/([a-z0-9-]+)-episode-([0-9]+)")) {
-		ShowData.title = document.querySelector("div.category a").textContent;
-
+		ShowData.title =
+			document.querySelector("div.category a")?.textContent ??
+			JSON.parse(
+				document
+					.querySelector('[class="yoast-schema-graph"]')
+					.innerHTML.replace(/@/gm, "")
+			).graph[3].itemListElement[1].name.replace(/Episode [0-9]*/gm, "");
```
Comment (Slowlife01) [inline websites/D/DramaCool/presence.ts:80]: And now this should not be necessary

Comment (DarkVillager) [reply websites/D/DramaCool/presence.ts:80]: Easiest way to get the title that doesn't change.

File: websites/D/DramaCool/presence.ts (hunk 2 above)
Comment (DarkVillager) [reply websites/D/DramaCool/presence.ts:56]: resolves by using covers?

**Review by Bas950 — APPROVED** [review]
